<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoldForm</class>
 <widget class="QWidget" name="MoldForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="mold::InvisibleClickableButton" name="myLovelyButton">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>175</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="warning_mold">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>145</y>
     <width>220</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;WARNUNG!&lt;/span&gt; Nächste Lüftung&lt;br/&gt;in: 50 min&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="mold::StatusBarForm" name="statusBar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>215</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="mold::RoomHeaderForm" name="header" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="controller" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>195</y>
     <width>320</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="controllerName">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true">Vent.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="actor_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true">Vent.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="actor_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true">Window 1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="actor_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true">Window 2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="actor_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true">Window 3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="windowOpen" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>155</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/settings/icons/window_open.svg);</string>
   </property>
  </widget>
  <widget class="QWidget" name="windowClosed" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>155</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/settings/icons/window_closed.svg);</string>
   </property>
  </widget>
  <zorder>statusBar</zorder>
  <zorder>myLovelyButton</zorder>
  <zorder>header</zorder>
  <zorder>controller</zorder>
  <zorder>warning_mold</zorder>
  <zorder>windowOpen</zorder>
  <zorder>windowClosed</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mold::InvisibleClickableButton</class>
   <extends>QPushButton</extends>
   <header>gui/InvisibleClickableButton.hpp</header>
  </customwidget>
  <customwidget>
   <class>mold::StatusBarForm</class>
   <extends>QWidget</extends>
   <header>gui/StatusBarForm.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mold::RoomHeaderForm</class>
   <extends>QWidget</extends>
   <header>gui/RoomHeaderForm.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
